gap/Parser.gi

```diff
@@ -585,27 +585,30 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
             if not IsBound( tree!.TitlePage.TitleComment ) then
                 tree!.TitlePage.TitleComment := [ ];
             fi;
-            tree!.content := tree!.TitlePage.TitleComment;
+            current_item := tree!.TitlePage.TitleComment;
 
-            Add( tree, "<URL>" );
-            Add( tree, current_command[ 2 ] );
-            Add( tree, "</URL>" );
+            Add( current_item, "<URL>" );
+            Add( current_item, current_command[ 2 ] );
+            Add( current_item, "</URL>" );
         end
     );
     title_item_list := [ "Title", "Subtitle", "Version", "TitleComment", "Author",
                          "Date", "Address", "Abstract", "Copyright", "Acknowledgements", "Colophon" ];
```

Comment:
And while I am nitpicking about things only partially related to this pull request: I think it would be really good if the manual contained an example showing of how to use these commands.